### ISSUE TITLE
Request: FreeCommander support

### ISSUE THREAD

**Issue by glocalglocal** [QL-Win/QuickLook#652]
Would you consider supporting FreeCommander out of the box please? This file manager uses Windows Preview to offer its own internal View when you press F3. I changed the key to <space> but (a) this viewer is slower to launch (in return for more functionality) (b) without QuickLook you don't get a viewer for the desktop or open/save dialogue. (c) Freecommander's viewer supports plugins but from what I see there is little choice. 
www.freecommander.com

**Comment by xupefei** [comment]
I didn't see FreeCommander provides an API for obtaining the selected file, so I am afraid that F3 is the only solution unless the author of FreeCommander can make some changes.

**Comment by glocalglocal** [comment]
Actually, it was easier than I thought. In FreeCommander Tools | Settings... | Programs | External viewers I specified QuickLook.exe as the external viewer for *.* (parameter "%ActivSel%"). Then, in Tools | Settings... | Define keyboard shortcuts... I specified <space> for "View external" and <Shift+Space> for "View" (ie the default internal viewer based on Windows Preview). And it works just fine inside FreeCommander, on the desktop, in Windows Explorer and in Windows Open/Save dialogues.
